bin/index.cjs

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
+const { execFileSync } = require('child_process');
+const shell = require('shelljs');
+
+const SUCCESS_CODE = 0;
+const ERROR_CODE = 1;
+
+function exitOnError(errorMessage) {
+  shell.echo(prettifyErrorMessage(errorMessage));
+  shell.exit(ERROR_CODE);
+}
+
+function prettifyStepTitle(title) {
+  return chalk.blue.bold(title);
+}
+
+function prettifyErrorMessage(errorMessage) {
+  return chalk.bgRed.bold(errorMessage);
+}
+
+if (!shell.which('git')) {
+  exitOnError('Sorry, this script requires git');
+}
+
+if (!shell.which('node') || !shell.which('npm')) {
+  exitOnError('Sorry, this script requires node & npm');
+}
+
+shell.echo(prettifyStepTitle('1/3 Cloning respository'));
+let cloneCommand = '';
+if (!shell.which('degit')) {
```

Comment:
What benefit do we keep if we have both? 🤔

Reply:
Not much, just that if someone has degit installed already, they could[ leverage it](https://github.com/Rich-Harris/degit#wait-isnt-this-just-git-clone---depth-1) to slightly faster download the repo. But, for the sake of simplicity, I'll leave just the git clone version